app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -20,10 +22,10 @@
   <div class="fw-bold fs-s mod-center-section-title resources">RESOURCES</div>
   <div id="mod-center-other-options">
     <a href="#" class="crayons-link crayons-link--block">
-      Connect Chat
-    </a>
-    <a href="#" class="crayons-link crayons-link--block">
-      Settings
+      Feedback?
```

Comment:
This link is supposed to open up the moderator's Team Channel, but I need help with the query to generate this link 🙏 @Zhao-Andy

Reply:
Hmm yeah, I think there's not really a good way yet... Might want to do something like:
```
# controller
@community_mod_channel = current_user.chat_channels.find_by("name LIKE ?", "Community Mods: Team%")

# view
<% if @community_mod_channel %> # fallback in case they're not in a channel, since we don't automate this yet
  <a href="<%= @community_mod_channel.path %>">Feedback? Share in the Community Mod channel!</a>
<% end %>
```